internal/resp_assertions/floating_point_bulk_string_assertion.go

```diff
@@ -0,0 +1,44 @@
+package resp_assertions
+
+import (
+	"fmt"
+	"strconv"
+
+	resp_value "github.com/codecrafters-io/redis-tester/internal/resp/value"
+)
+
+type FloatingPointBulkStringAssertion struct {
+	ExpectedValue float64
+	Tolerance     float64
+}
+
+func NewFloatingPointBulkStringAssertion(expectedValue float64, tolerance float64) RESPAssertion {
+	return FloatingPointBulkStringAssertion{
+		ExpectedValue: expectedValue,
+		Tolerance:     tolerance,
+	}
+}
+
+func (a FloatingPointBulkStringAssertion) Run(value resp_value.Value) error {
+	if value.Type != resp_value.BULK_STRING {
+		return fmt.Errorf("Expected bulk string, got %s", value.Type)
+	}
+
+	stringValue := value.String()
+	f, err := strconv.ParseFloat(stringValue, 64)
```

Comment:
```suggestion
	floatValue, err := strconv.ParseFloat(stringValue, 64)
```

Avoid single-letter variable names in general, only exception is struct method target (like `a` here), or `i`/`j` in a loop